simulation/agents.py

```diff
@@ -226,10 +226,48 @@ def get_quote_insulation_elements(
 
         return insulation_quotes
 
+    def choose_n_elements_to_insulate(self):
+
+        # Derived from the VERD Project, 2012-2013. UK Data Service. SN: 7773, http://doi.org/10.5255/UKDA-SN-7773-1
+        # Based upon the choices of houses in 'Stage 3' - finalising or actively renovating
+
+        return random.choices([1, 2, 3], weights=[0.76, 0.17, 0.07])[0]
+
+    def choose_insulation_elements(
+        self, insulation_quotes: Dict[Element, float], n_elements: int
+    ) -> Dict[Element, float]:
+
+        sorted_tuples = sorted(insulation_quotes.items(), key=lambda item: item[1])
+        quotes_sorted = {k: v for k, v in sorted_tuples}
+
+        return {k: quotes_sorted[k] for k in list(quotes_sorted.keys())[:n_elements]}
+
+    def install_insulation_elements(
+        self, insulation_elements: Dict[Element, float]
+    ) -> None:
+
+        for element in insulation_elements:
+            if element == Element.ROOF:
+                self.roof_energy_efficiency = 5
```

Comment:
Do you think we should stick with efficiencies as integers, or instead add an `Efficiency(enum.Enum)` with `VERY_LOW, LOW, AVERAGE, HIGH, VERY_HIGH` ?

Reply:
I think it's fine with the integers atm